client/src/components/ProfilePicture/NewUserUploadPicture.css

```diff
@@ -0,0 +1,22 @@
+.NewUserUpload {
+  height: 233px;
+  width: 233px;
+  border: solid var(--yellow) 1px;
+  border-radius: 50%;
+  text-align: center;
+  /* display: flex;
+  flex-direction: column;
+  justify-content: center; */
+}
+
+.NewUserUpload__p {
```

Comment:
If this is just a `p` tag, then just use the descendent selector: `.NewUserUpload p`